### PR TITLE
Update client settings

### PR DESCRIPTION
**Pull request by khorne3**
https://auth0-docs-content-pr-4614.herokuapp.com/docs/clients/client-settings/native

### CODE REVIEW

**Comment by khorne3** [comment]
@fiddur I'm assuming the subdomains clause isn't affected by the limitations on path/querystrings/hashes?

**Comment by fiddur** [comment]
@khorne3 Right, subdomains should still work as intended.

File: articles/clients/client-settings/_settings-pt2.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
...
-- **Allowed Origins (CORS)**: Set of URLs that will be allowed to make requests from JavaScript to Auth0 API (typically used with CORS). This prevents same-origin policy errors when using Auth0 from within a web browser. By default, all your callback URLs will be allowed. This field allows you to enter other origins if you need to. You can specify multiple valid URLs by comma-separating them. You can use the star symbol as a wildcard for subdomains (`*.google.com`). Notice that querystrings and hash information are not taking into account when validating these URLs.
+- **Allowed Origins (CORS)**: Set of URLs that will be allowed to make requests from JavaScript to Auth0 API (typically used with CORS). This prevents same-origin policy errors when using Auth0 from within a web browser. By default, all your callback URLs will be allowed. This field allows you to enter other origins if you need to. You can specify multiple valid URLs by comma-separating them. You can use the star symbol as a wildcard for subdomains (`*.google.com`). Notice that paths, querystrings and hash information are not taking into account when validating these URLs (and may, in fact, cause the match to fail).
```
Comment (fiddur) [inline articles/clients/client-settings/_settings-pt2.md]: `are not taken into account` (not `taking`) I think is more proper English.

**Review by fiddur — COMMENTED** [review]
The updated text looks good to me.  I don't know what the other stuff is about :)

**Pull request merged**